code-challenges.test.js

```diff
@@ -12,56 +12,122 @@
-  thursday: 20,
-  friday: 15,
-  saturday: 65
-}
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
-
-// b) Create the function that makes the test pass.
-
-// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
-// a) Create a test with expect statements for each of the variables provided.
+// describe("sorter", () => {
+//   it("sorted from least to greatest", () => {
+//     expect(sorter(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+//     expect(sorter(studyMinutesWeek2)).toEqual([10, 15, 18, 20, 45, 60, 65])
+//   })
+// })
+
+
+// const studyMinutesWeek1 = {
+//   sunday: 90,
+//   monday: 30,
+//   tuesday: 20,
+//   wednesday: 15,
+//   thursday: 30,
+//   friday: 15,
+//   saturday: 60
+// }
+// // Expected output: [15, 15, 20, 30, 30, 60, 90]
+
+// const studyMinutesWeek2 = {
+//   sunday: 18,
+//   monday: 10,
+//   tuesday: 45,
+//   wednesday: 60,
+//   thursday: 20,
+//   friday: 15,
+//   saturday: 65
+// }
+// // Expected output: [10, 15, 18, 20, 45, 60, 65]
+
+// // b) Create the function that makes the test pass.
+// // pseudo code :input an object 
+//   // Output and ordered array from least to greatest 
+//   // create a function that can take and turn an Object into an array, using the Object.Values(array) .
+//   // and then sorting them from least to greatest using the .sort
+
+// const sorter = (array)=> {
+// return Object.values(array).sort()
+  
+// }
+
+
+// // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+// describe("accumulatedSum", () => {
+//   it("add it all up ", () => {
+//     expect(accumulatedSum(accountTransactions1)).toEqual([100, 83, 60, 51])
+//     expect(accumulatedSum(accountTransactions2)).toEqual([250, 161, 261, 165])
+//     expect(accumulatedSum(accountTransactions3)).toEqual([])
+//   })
+// })
+// // a) Create a test with expect statements for each of the variables provided.
+
+
+
+// const accountTransactions1 = [100, -17, -23, -9]
+// // Expected output: [100, 83, 60, 51]
+
+// const accountTransactions2 = [250, -89, 100, -96]
+// // Expected output: [250, 161, 261, 165]
+
+// const accountTransactions3 = []
+// // Expected output: []
+
+// // b) Create the function that makes the test pass.
+
+// // pseudo code: 
+// // Input and array of numbers
+// // Output: the total sum of all the numbers inside of a array.
+// // use the .reduce method and return in a new array
 
-const accountTransactions1 = [100, -17, -23, -9]
-// Expected output: [100, 83, 60, 51]
 
-const accountTransactions2 = [250, -89, 100, -96]
-// Expected output: [250, 161, 261, 165]
+// const accumulatedSum = (array)=>{
+// var new_array = [];
+//  array.reduce(function(a,b,i) { return new_array[i] = a+b; },0);
+// return new_array
 
-const accountTransactions3 = []
-// Expected output: []
+// }
 
```

Comment:
Awesome job with this function and getting the output you need! 👏🏽 One of the things that will help your pseudocode is talking about what each piece of the function is doing. For example, you can reference a+b and how that will add two elements.